### PR TITLE
Add a new section for ontology information and improve test cases

### PR DESCRIPTION
**Pull request by NutthanichN**
### Test case 1 (`test_case1.yaml`)
- Added a new section for collecting general information of an ontology
- Added annotation properties from Dublin Core in `owl:AnnotationProperty` section
- Edited .md file to match .yaml file
- Fixed syntax error in `test_case1.yaml` (thx @theethaj who pointed this out) by grouping a subclass with necessary conditions

Before
```
rdfs:subClassOf:
  - NamedPizza
  owl:Restriction:
    - hasTopping only (MozzarellaTopping or TomatoTopping)
    - hasTopping some MozzarellaTopping
    - hasTopping some TomatoTopping
```

After
```
rdfs:subClassOf:
  - NamedPizza
  - hasTopping only (MozzarellaTopping or TomatoTopping)
  - hasTopping some MozzarellaTopping
  - hasTopping some TomatoTopping
```

### Test case 2 (`test_case2.yaml`)
- Added custom datatype in `rdfs:Datatype` section of `test_case2.yaml`
- Edited .md file to match .yaml file

### CODE REVIEW

File: tests/test_cases/test_case2.md [hunk 1]
```diff
@@ -313,9 +310,14 @@ Subproperty Of:
...
-- Literal
-- String
+
+## personAge
+### Annotations
+Comment: Age of person which is the integer between 0 to 150
+
+### Description
+Datatype Definitions:
+  - integer[>=0, <=150]
```
Comment (MaiMee1) [inline tests/test_cases/test_case2.md:320]: There can be many descriptions?

File: tests/test_cases/test_case1.md [hunk 2]
```diff
@@ -180,8 +191,15 @@ Subclass Of:
...
-- Label
+## title
+### Annotations
+Comment: A name given to the resource
+
+## licence
+### Annotations
+Comment: A legal document giving official permission to do something with the resource.
+
+Lable: License
```
Comment (MaiMee1) [inline tests/test_cases/test_case1.md:202]:
```suggestion
Label: License
```
I think you have the wrong spelling?

File: tests/test_cases/test_case1.md [hunk 3]
```diff
@@ -1,3 +1,15 @@
+# Ontology description
+## Annotations
+### lable
+  - String
+
+### title
+  - Literal
```
Comment (MaiMee1) [inline tests/test_cases/test_case1.md:7]: Should these be "pizza"?
// Also another spelling mistake here

**Review by MaiMee1 — CHANGES_REQUESTED** [review]

File: tests/test_cases/test_case2.md (hunk 1 above)
Comment (Wijantra) [reply tests/test_cases/test_case2.md:320]: Protege allows many descriptions but we are not sure that is it cause error ontology. We do not encourage to many descriptions so we will use one description

File: tests/test_cases/test_case1.md (hunk 2 above)
Comment (Wijantra) [reply tests/test_cases/test_case1.md:202]: yes i have wrong spelling
i will change it

File: tests/test_cases/test_case1.md (hunk 3 above)
Comment (Wijantra) [reply tests/test_cases/test_case1.md:7]: yes i will change it

File: tests/test_cases/test_case2.md (hunk 1 above)
Comment (Wijantra) [reply tests/test_cases/test_case2.md]: I changed it

File: tests/test_cases/test_case1.md (hunk 2 above)
Comment (Wijantra) [reply tests/test_cases/test_case1.md]: I changed it

File: tests/test_cases/test_case1.md (hunk 3 above)
Comment (Wijantra) [reply tests/test_cases/test_case1.md]: I changed it

File: tests/test_cases/test_case2.md (hunk 1 above)
Comment (MaiMee1) [reply tests/test_cases/test_case2.md]: Looks like the only thing we haven't accounted for is an (Enumerated Datatype)[https://www.w3.org/TR/2004/REC-owl-ref-20040210/#EnumeratedDatatype], but maybe we could do that later.

**Review by MaiMee1 — APPROVED** [review]

**Pull request merged**